docker-compose.manager.yml

```diff
@@ -0,0 +1,55 @@
+services:
+  admin-dashboard:
+    image: ragapp/admin-dashboard
+    build: src/manager
+    command:
+      - fastapi
+      - run
+      - main.py
+      - --port
+      - '80'
+      - --root-path
+      - /dashboard
+    ports:
+      - '8100:80'
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock:ro
```

Comment:
I'm mounting docker.sock directly to the dashboard app with readonly permission, it's working now. But once we need start/stop/remove the container then we should use a dedicated `socat` container for that (https://github.com/alpine-docker/multi-arch-docker-images/tree/master/socat)